src/app/charts/charts.component.ts

```diff
@@ -323,4 +324,8 @@ export class ChartsComponent implements OnInit {
         console.info('Background click')
     }
 
+    log(msg: string) {
```

Comment:
Is this menat to stay or shoul dbe removed?

Reply:
The intention was to keep it to demonstrate the click event in the HTML element inside the SVG